dark/proteins.py

```diff
@@ -23,6 +24,75 @@
 from dark.html import NCBISequenceLinkURL
 from dark.reads import Reads
 
+# The following regex is deliberately greedy (using .*) to consume the
+# whole protein title before backtracking to find the last [virus title]
+# section. That way, it will match just the last [virus title] in a
+# protein. This avoids situations in which two [...] delimited substrings
+# are present in a protein name (in which case we just want the last).
+# E.g., the following is a complete protein name:
+#
+#   gi|19919894|ref|NP_612577.1| Enzymatic polyprotein [Contains: Aspartic
+#   protease; Endonuclease; Reverse transcriptase] [Carnation etched ring
+#   virus]
+#
+# Unfortunately the regex doesn't find the virus title when the protein
+# title has nested [...] sections, as in this example:
+#
+#   gi|224808893|ref|YP_002643049.1| replication-associated protein [Tomato
+#   leaf curl Nigeria virus-[Nigeria:2006]]
+#
+# I decided not to worry about nested [...] sections (there are only 2
+# instances that I know of).
+_VIRUS_RE = re.compile('^(.*)\[([^\]]+)\]$')
+
+# The virus title assigned to proteins whose title strings cannot be parsed
+# for a virus title (see previous comment).  Do not use '<', '>' or any
+# other HTML special chars in the following.
+_NO_VIRUS_TITLE = '[no virus name found in protein title]'
+
+
+def splitTitles(titles):
+    """
+    Split a title like "Protein name [virus name]" into two pieces using
+    the final square brackets to delimit the virus name.
```

Comment:
Here you talk about `virus name`, rather than `virus title`. 
I'm not sure I like the `virus title`, mainly because we use `title` in many places with different meanings. How about using `virus name` throughout?